02_configure_host.sh

```diff
@@ -83,6 +83,7 @@ ansible-playbook \
     -e "num_masters=$NUM_MASTERS" \
     -e "num_workers=$NUM_WORKERS" \
     -e "num_extraworkers=$NUM_EXTRA_WORKERS" \
+    -e "num_fleetings=$NUM_FLEETINGS" \
```

Comment:
Do we need a new node type for this, or should the install ISO be booted in-place on one of the masters (to replicate what customer cluster installs would do)?

Reply:
I don't have a strong opinion on it, it just seemed like the easiest way to integrate at the time.